python-sdk/nuscenes/scripts/export_poses.py

```diff
@@ -0,0 +1,206 @@
+# nuScenes dev-kit.
+# Code contributed by jean-lucas, 2020.
+
+"""
+Exports the nuScenes ego poses as "GPS" coordinates (lat/lon) for each scene into JSON or KML formatted files.
+"""
+
+
+import argparse
+import json
+import math
+import os
+from typing import List, Tuple, Dict
+
+from tqdm import tqdm
+
+from nuscenes.nuscenes import NuScenes
+
+
+EARTH_RADIUS_METERS = 6.378137e6
+REFERENCE_COORDINATES = {
```

Comment:
how are these obtained?

Reply:
These can be computed from the slippy map tile system, which tiles the globe into a grid at a specified resolution: https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames